apps/aecontract/src/aect_contracts.erl

```diff
@@ -513,7 +513,6 @@ assert_field_store(store_k = Field, X, #{vm := VM} = Version) when is_binary(X),
     catch _:_ -> error({illegal, Field, X}) end;
 assert_field_store(store_k, X, #{vm := VM}) when is_binary(X),
                                                  byte_size(X) > 0,
-                                                 byte_size(X) =< 32,
```

Comment:
I just checked https://github.com/aeternity/protocol/blob/master/serializations.md#contract and the upper size is indeed not defined.